types/esri-leaflet/index.d.ts

```diff
@@ -596,6 +611,148 @@ declare module 'leaflet' {
+         */
+        interface ImageMapLayerOptions extends ImageOverlayOptions, LayerOptionsBase {
+            /**
+             * If there are multiple bands, you can specify which bands to export
+             */
+            bandIds?: string | undefined;
+
+            /**
+             * The pixel value representing no information.
+             */
+            noData?: number | undefined;
+
+            /**
+             * Interpretation of the noData setting.
+             */
+            noDataInterpretation?: string | undefined;
+
+            /**
+             * Leave pixelType as unspecified, or UNKNOWN, in most exportImage use cases, unless such pixelType is
+             * desired. Possible values: C128, C64, F32, F64, S16, S32, S8, U1, U16, U2, U32, U4, U8, UNKNOWN.
+             */
+            pixelType?: string | undefined;
+
+            /**
+             * A JSON representation of a raster function
+             */
+            renderingRule?: object | undefined;
+
+            /**
+             * A JSON representation of a mosaic rule
+             */
+            mosaicRule?: object | undefined;
+        }
+
+        /**
+         * Render and visualize Image Services from ArcGIS Online and ArcGIS Server.
+         * Image Services provide access to raster data through a web service.
+         */
+
+        class ImageMapLayer extends RasterLayer {
+            constructor(options: ImageMapLayerOptions);
+
+            /**
+             * Uses the provided function to create a popup that will identify pixel value(s) whenever the map is clicked.
+             * Your function will be passed an object with a `pixel` property that is a GeoJSON Point with the pixel
+             * value(s) at the clicked location and should return the appropriate HTML. If you do not want to open the
+             * popup when there are no results, return false.
+             *
+             * NOTE: by default, if the layer has a mosaic rule applied, then the same rule will be applied to the
+             * identify request. Conversely, if the layer has a rendering rule applied, that rule is NOT applied to the
+             * layer so that the raw pixel value can be returned. If you need specific control over how these rules
+             * (and/or other identify parameters) are passed to the identify service, use L.esri.IdentifyImage.
+             */
+            bindPopup(
+                fn: LayerCallbackHandler | ((layer: Layer) => Content) | string | HTMLElement | Popup,
+                popupOptions?: PopupOptions,
+            ): this;
+
+            /**
+             * Removes a popup previously bound with bindPopup.
+             */
+            unbindPopup(): this;
+
+            /**
+             * Returns the current band value(s).
+             */
+            getBandIds(): string;
+
+            /**
+             * 	Specify a single band to export, or you can change the band combination (red, green, blue) by
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/esri-leaflet/index.d.ts;
  Line: 685;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.